truetype/truetype.go

```diff
@@ -95,11 +99,16 @@ type cm struct {
 	start, end, delta, offset uint32
 }
 
+// An nr holds a font's name, style
+type nr struct {
+	fontInfo [2]string
```

Comment:
Is "fontInfo" a truetype concept? Why is it [2]string and not:

type nameRecord struct {
  name, style string
}